conf/config-default.yaml

```diff
@@ -170,7 +170,8 @@ nginx_config:                     # Config for render the template to generate n
   stream:
     enable_access_log: false                 # Enable stream proxy access logging.
     access_log: logs/access_stream.log       # Location of the stream access log.
-    access_log_format: "$remote_addr [$time_local] $protocol $status $bytes_sent $bytes_received $session_time" # Customize log format: http://nginx.org/en/docs/varindex.html
+    access_log_format: |
+      "$remote_addr [$time_local] $protocol $status $bytes_sent $bytes_received $session_time" # Customize log format: http://nginx.org/en/docs/varindex.html
```

Comment:
due to this change, are you sure the comment won't be parsed as a part of the string?

Reply:
This change has nothing to do with this modification. If you think it is necessary, it is recommended to open a new PR to deal with it.

Reply:
@Revolyssup please address this comment.